README.md

```diff
@@ -210,13 +251,45 @@ commands:
     ## Or build the site and run all tests
     make all
 
-#### Partial build for faster preview
+#### Fast Partial Previews Or Builds
 
 In order to preview some changes faster, you can disable all plugins and
-languages but those you need by prefixing the `ENABLED_LANGS` and `ENABLED_PLUGINS`
-environment variables:
+languages except those you need by prefixing the `ENABLED_LANGS` and
+`ENABLED_PLUGINS` environment variables to your command line.  For
+example, do this to disable everything:
+
+    ## Fast preview, takes less than 30 seconds
+    ENABLED_PLUGINS="" ENABLED_LANGS="" make preview
 
-    ENABLED_PLUGINS="events autocrossref" ENABLED_LANGS="en fr" make all
+    ## Fast build and tests, takes less than 50 seconds
+    ## Some tests may fail in fast mode; use -i to continue despite them
+    ENABLED_PLUGINS="" ENABLED_LANGS="" make -i valid
+
+Then to enable some plugins or languages, you can add them back in.
+For example:
+
+    ## Slower (but still pretty fast) build and test
+    ENABLED_PLUGINS="events autocrossref" ENABLED_LANGS="en fr" make -i valid
+
+Plugins include:
+
+| Plugin       | Seconds | Remote APIs    | Used For
+|--------------|---------|----------------|------------------------
+| alerts       | 5       | --             | Network alert pages
+| autocrossref | 90      | --             | Developer documentation
+| contributors | 5       | GitHub.com     | Contributor listings
+| events       | 5       | Meetup.com; Google Maps | Events page
+| glossary     | 30      | --             | Developer glossary
+| redirects    | 20      | --             | Redirects from old URLs
+| releases     | 10      | --             | Bitcoin Core release notes; Download page
+| sitemap      | 0       | --             | /sitemap.xml
```

Comment:
@harding FWIW, the run time for this plugin is influenced by the number of languages enabled. With no language disabled, I can see a 10 seconds difference, in case you'd like to update the number.